.teamcity/_self/lib/e2e/Steps.kt

```diff
@@ -0,0 +1,179 @@
+package _self.lib.e2e
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildSteps
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.ScriptBuildStep
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+
+/**
+ * Prepares the environment to run Playwright tests.
+ *
+ * This will generate a Script step that:
+ *
+ * - Updates node
+ * - Set some env variables
+ * - Installs dependencies for @automattic/calypso-e2e and builds it
+ * - Installs dependencies for wp-e2e-tests
+ *
+ * It uses Yarn focus mode (https://yarnpkg.com/cli/workspaces/focus) to minimize the dependencies
+ * installed and save some time. Yarn may throw some warnings about unused overrides, but they are safe to ignore.
+ */
+fun BuildSteps.prepareEnvironment(): ScriptBuildStep {
+	return script {
+		name = "Prepare environment"
+		scriptContent = """
```

Comment:
Can we use `bashNodeScript` instead of repeating bash options and the command to update node?

Reply:
Now using `bashNodeScript`.